src/main/java/com/google/sps/data/Challenges.java

```diff
@@ -0,0 +1,41 @@
+
+package com.google.sps.data;
+
+import java.util.Arrays;
+import java.util.List;
+
+
+public final class Challenges {
```

Comment:
I'm not sure why this class is separate from the `Challenge` class. It would make sense to put all these constants into the `Challenge` class and remove this class.

Reply:
I have to agree with the above, this class does not seem to add functionality more than instantiating the Challenge class. Could this static final data be incorporated into the Challenge class or could this be instantiated where it is needed (ie. the Challenges servlet at init)?